dev-lang/ispc/ispc-9999.ebuild

```diff
@@ -35,9 +35,9 @@ DEPEND="
 	"
```

Comment:
I think bison/flex are BDEPEND (you don't need to run them for CHOST during cross compilation).

Also, if we definitely NEED clang/llvm (cannot use gcc?), we should check in pkg_setup and bail out if not using those.

Reply:
It does work with gcc too. It is just not supported.

There was an issue before were clang only compile flags were used and it would prevent gcc from not erroring out.
But I reported it and it is not fixed upstream: https://github.com/ispc/ispc/pull/1829
(Fixed in the 1.14.0 version and the live version).

I'm using gcc to compile this on my side. So clang is not strictly needed. (llvm is though)

Reply:
OK.